tests/test_pytomorrowio.py

```diff
@@ -335,3 +295,40 @@ async def test_timelines_realtime_nowcast_hourly_daily(aiohttp_client, mock_url)
         assert isinstance(forecast, Sequence)
         print(key, len(forecast))
         assert len(forecast) == expected_count
+
+
+async def test_errors(aiohttp_client):
+    """Test errors."""
+    with pytest.raises(ValueError):
+        TomorrowioV4("bogus_api_key", 0, 0, "fake_unit")
+
+    with pytest.raises(InvalidTimestep):
+        TomorrowioV4.available_fields("not_a_timestep", [])
+
+    session = await create_session(
+        aiohttp_client, "timelines_realtime.json", status=HTTPStatus.BAD_GATEWAY
```

Comment:
Use bogus file name to demonstrate that it never gets that far.
```suggestion
        aiohttp_client, "bogus_file.json"
```